tests/src/Functional/AspNetCore/AspNetCore.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <NativeCodeGen>readytorun</NativeCodeGen>
+  </PropertyGroup>
+
+  <!--
+    API to skip compilation for assemblies CPAOT cannot successfully compile yet
+  -->
+  <ItemGroup>
+    <ReadyToRunSkipAssembly Include="System.Private.CoreLib" />
+    <ReadyToRunSkipAssembly Include="System.Net.Http" />
```

Comment:
Nit - System.Net.Http should be fine now. S.P.Corelib still needs my latest change pending PR.

Reply:
It compiles fine but the test fails to run properly, so we'll need to take a deeper look.